core/network/src/sync.rs

```diff
@@ -144,13 +143,22 @@ impl<B: BlockT> ChainSync<B> {
 				(Ok(BlockStatus::Unknown), b) if b == As::sa(0) => {
```

Comment:
`b.is_zero()` is preferable (though you'll need to `use primitives::traits::Zero;`